test/integration/load_node_test.cpp

```diff
@@ -78,6 +79,45 @@ TEST(LoadNodeTest, IterateMap) {
   EXPECT_EQ(3, i);
 }
 
+static void alias_access(Node&& doc) {
+  Node A = doc["A"];
+  Node B = doc["B"];
+
+  // A and B have the same content
+  ASSERT_TRUE(A);
+  ASSERT_TRUE(A.IsMap());
+  ASSERT_TRUE(B);
+  ASSERT_TRUE(B.IsMap());
+
+  // A and B have the same content
+  std::map<std::string, std::string> values = {{"str", "string"}, {"float", "3.1415"}, {"int", "42"}};
+  for (YAML::const_iterator it = A.begin(); it != A.end(); ++it) {
+    const std::string& key = it->first.as<std::string>();
+    SCOPED_TRACE("key " + key);
+    Node a = A[key];
+    Node b = B[key];
+    EXPECT_TRUE(a);
+    EXPECT_TRUE(b);
+    EXPECT_TRUE(a.IsScalar());
+    EXPECT_TRUE(b.IsScalar());
+
+    // a and b should be identical
+    EXPECT_STREQ(a.as<std::string>().c_str(), b.as<std::string>().c_str());
+    // ... and have the values given in the map
+    EXPECT_STREQ(a.as<std::string>().c_str(), values[key].c_str());
+  }
+}
+
+TEST(LoadNodeTest, AliasAccess) {
+  alias_access(Load("{A: &DEFAULT {str: string, int: 42, float: 3.1415}, B: *DEFAULT}"));
+}
+
+TEST(LoadNodeTest, AliasAccessStream) {
+  std::ifstream input("integration/alias.yaml");
```

Comment:
No need to have this separate file-based loading; it causes various problems like worrying about working directories, etc.

Instead, if you're worried about flow vs. block, then you can keep the alias_access function and have two tests; otherwise, one test is fine.

Reply:
Feel free to to adapt and use only what you need.
While tracking down the issue, I tried  variation along many directions. One was to load from an input...

Reply:
Do you mean you're abandoning this PR?

Reply:
If you don't think, it's useful, yes.

Reply:
I do think it's useful, and I added some comments before approval.